app/client/ui/Tools.ts

```diff
@@ -97,12 +105,15 @@ export function tools(owner: Disposable, gristDoc: GristDoc, leftPanelOpen: Obse
       cssPageButton(cssPageIcon('Log'), cssLinkText(t("Document History")), testId('log'),
         dom.on('click', () => gristDoc.showTool('docHistory')))
     ),
-    dom.maybe(docPageModel.isFork, () => {
+    dom.maybe(
```

Comment:
```suggestion
    dom.maybe(trunkAcceptsProposals
```

`use => use....` is not needed here.